projects/igniteui-angular/src/lib/services/overlay/overlay.spec.ts

```diff
@@ -2753,11 +2735,12 @@ describe('igxOverlay', () => {
             overlayWrapper.addEventListener('keydown', (event: KeyboardEvent) => {
                 if (event.key === targetButton) {
                     overlayWrapper = document.getElementsByClassName(CLASS_OVERLAY_WRAPPER_MODAL)[0];
+                    expect(overlayWrapper).toBeFalsy();
```

Comment:
I really do not like this way of testing. You can check the count of the overlays attached to the document